.github/scripts/bot-assignment-check.sh

```
@@ -36,20 +43,20 @@ issue_has_gfi() {
-    # For triage users, exclude issues with 'mentor-duty' label
+    echo "Triage user detected — excluding ${MENTOR_LABEL} issues from count." >&2
+    # For triage users, exclude issues with the configured mentor label
      gh api "repos/${REPO}/issues?per_page=100&page=1" \
        -f assignee="${ASSIGNEE}" \
        -f state=open \
-       --jq '.[]
+       --jq --arg mentor_label "$MENTOR_LABEL" '.[]
             | select(.pull_request == null)
-            | select(any(.labels[]; .name == "mentor-duty") | not)
+            | select(any(.labels[]; .name == $mentor_label) | not)
             | .number' | grep -c . || echo 0
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)